src/lib/_imports/trumps/s-hero-banner/s-hero-banner.demo.css

```diff
@@ -0,0 +1,11 @@
+body > main {
+    margin-inline: unset;
+    padding-block: unset;
+    width: unset;
+}
+  
```

Comment:
**🙋‍♂️ Too Much CSS?** I think you can delete all this. Do correct me if I'm wrong.

```suggestion
```

| too much css? | just enough css ? | not enough css… |
| - | - | - |
| ![1 - too much css](https://github.com/TACC/Core-Styles/assets/62723358/67ef6c8f-1372-48ec-a834-e3e74c0cbaa7) | ![2 - just enough css](https://github.com/TACC/Core-Styles/assets/62723358/08f69078-8716-439b-baa8-2921a3495095) | ![3 - not enough css](https://github.com/TACC/Core-Styles/assets/62723358/aca3da16-a700-44d7-ade1-3e2d662025ac) |

Reply:
Fixed